certbot/CHANGELOG.md

```diff
@@ -14,7 +14,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 
 ### Fixed
 
-*
+* Fix incompatibility (TypeError) with lexicon >= v3.6.0, but breaks
+  compatibility with lexicon < v3.1.0
```

Comment:
In fact, I think it is more important to say that Lexicon >= 3.1.0 is now required, and optionally say that it is to fix compatiblity with 3.6.0 and higher. The lower bound is the info you want to keep in the long term.